javascript/javascript1/week2/script.js

```diff
@@ -0,0 +1,146 @@
+/*freecodecamp.org account name: mehmetnuricavdar
+https://www.freecodecamp.org/mehmetnuricavdar
+*/
+
+// ######    Flight booking fullname function
+
+function getFullname(firstname, surname) {
+  const fullName = `${firstname} ${surname}`;
+  return fullName;
+}
+
+let fullname1 = getFullname("Benjamin", "Hughes");
+let fullname2 = getFullname("Mehmet", "Cavdar");
+
+// ###### Formal fullname
+
+function getFullname(firstname, surname, useFormalName, gender) {
+  const fullName = `${firstname} ${surname}`;
+  if (useFormalName === true) {
```

Comment:
you don't need to write `useFormalName === true` if you just use it in your condition, it will be replaced by it's value which is true or false and it will look like this `if ( useFormalName )`

Reply:
Yes you are right, i missed that part.